compiler/generator/rust/rust_code_container.cpp

```diff
@@ -134,22 +134,24 @@ void RustCodeContainer::produceInternal()
     // Fill
     tab(n + 1, *fOut);
     string counter = "count";
+    tab(n + 1, *fOut);
+    *fOut << "#[allow(unused_unsafe)]";
     if (fSubContainerType == kInt) {
         tab(n + 1, *fOut);
-        *fOut << "pub fn fill" << fKlassName
-              << subst("(&mut self, $0: i32, table: &mut[i32]) {", counter);
+        *fOut << "fn fill" << fKlassName
+              << subst("(&mut self, $0: i32, mut table: *mut[i32]) { unsafe{", counter);
     } else {
         tab(n + 1, *fOut);
-        *fOut << "pub fn fill" << fKlassName
-              << subst("(&mut self, $0: i32, table: &mut[FaustFloat]) {", counter);
+        *fOut << "fn fill" << fKlassName
+              << subst("(&mut self, $0: i32, mut table: *mut[FaustFloat]) { unsafe{", counter);
```

Comment:
Is the `mut` before `table` actually needed?

At least my example was compiling without it, and as far as I can see the pointer itself isn't mutated, right?